tests/Makefile.common

```diff
@@ -855,14 +855,14 @@ ifdef ESC_ON
 ifndef NO_MERGE
 ifndef NO_CSHARP
 # ESC  MERGE  CSHARP
-txt-diff: txt-daikon-diff txt-chicory-diff txt-esc-diff inv-checker-diff \
-		  txt-jml-diff txt-merge-diff spinfo-static-diff txt-daikon-end2end-diff txt-chicory-diff \
-		  dyncomp-diff txt-simplify-diff txt-csharpcontract-diff txt-jaif-diff
+txt-diff: dyncomp-diff txt-daikon-diff txt-chicory-diff txt-esc-diff inv-checker-diff \
```

Comment:
I see why dyncomp-diff comes early:  if it differs, many other things are likely to differ too and so it should be shown first.

Why is txt-chicory-diff removed?  I think it could possibly change without affecting the Daikon output, and that would be good for tests to notice.

Reply:
These diff targets have a prerequisite chain that leads all the way back to running the requisite tool.  Putting dyncomp-diff first ensures that DynComp is run prior to Chicory.  The prerequisite 'txt-chicory-diff' was duplicated so I removed the second instance.